cmd/vic-machine/create/create.go

```diff
@@ -1062,7 +1069,7 @@ func (c *Create) generateCertificates(server bool, client bool) ([]byte, *certif
 
 	var certs []byte
 	// generate the certs and keys with names conforming the default the docker client expects
-	err := os.MkdirAll(fmt.Sprintf("./%s", c.certPath), 0700)
+	err := os.MkdirAll(fmt.Sprintf("%s", c.certPath), 0700)
```

Comment:
So is the certPath absolute now? Where this directory gets created may not be predictable if its not.